ast/compile.go

```diff
@@ -536,6 +539,38 @@ func (c *Compiler) GetRulesWithPrefix(ref Ref) (rules []*Rule) {
 	return rules
 }
 
+func (c *Compiler) GetPackageAnnotations(pkg *Package) *Annotations {
+	as := c.annotationSet
+
+	result := make([]*Annotations, 0, 2)
+
+	result = append(result, as.getSubpackagesScope(pkg.Path)...)
+
+	if x := as.getPackageScope(pkg); x != nil {
+		result = append(result, x)
+	}
+
+	return mergeAnnotationsList(result)
+}
+
+func (c *Compiler) GetRuleAnnotations(rule *Rule) *Annotations {
```

Comment:
Someone using OPA as a lib can use this and `GetPackageAnnotations()` to get access to the annotations for a given rule or package. Question is, is there some other abstraction layer where it would also make sense to add corresponding helper functions?
If not, I'll just document that these two functions are available, and how to use them.